pkg/virtual/apiexport/builder/build.go

```diff
@@ -116,8 +116,8 @@ func BuildVirtualWorkspace(
 
 				impersonationConfig := rest.CopyConfig(cfg)
 				impersonationConfig.Impersonate = rest.ImpersonationConfig{
-					UserName: "system:serviceaccount:default:rest",
-					Groups:   []string{bootstrap.SystemKcpAdminGroup},
+					UserName: authorizationbootstrap.SystemServiceAccountDefaultRest,
```

Comment:
This is changing the user name from `system:serviceaccount:default:rest` to `system:serviceaccount:default:kcp-rest`. What is the reason?

Reply:
would get rid of the constant. It is really just a random string.